### PR TITLE
Update forge world mappings

### PR DESCRIPTION
**Pull request by clienthax**

### CODE REVIEW

File: worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java [hunk 1]
```diff
@@ -238,7 +238,7 @@ public boolean regenerate(Region region, EditSession editSession) {
                 try {
                     u = ChunkProviderServer.class.getDeclaredField("field_73248_b"); // chunksToUnload
```
Comment (Lunatrius) [inline worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java]: Missed the comments. Anyway, wouldn't it be easier to just use:
```java
Field u = ReflectionHelper.findField("field_73248_b", "droppedChunksSet");
```

Comment (clienthax) [inline worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorld.java]: Possibly, however this works and is inline with the original code, i remember having issues with reflectionhelper screwing in the past.

**Comment by wizjany** [comment]
When were these renamed?

**Comment by clienthax** [comment]
Mcpbot says 2 years ago which seems a bit out of range..

**Comment by wizjany** [comment]
yea kinda doubt that since these were at least working on 1.7. i don't know if this changed at the same time as 1.8 or later though, because if it's working on one version but not another we should update deps and require the correct version

**Comment by clienthax** [comment]
This is only a issue when running in dev envs btw, normally it uses the obfuscated mappings

**Comment by TomyLobo** [comment]
2 years... that would be the [1.6 release] (http://minecraft.gamepedia.com/Versions)

**Comment by kenzierocks** [comment]
I guess I never used regenerate in development...

**Comment by kenzierocks** [comment]
If you rename the comments then I think this is good.

**Comment by zml2008** [comment]
Pretty sure we'd have to become a coremod to have an AT, which is really something we should avoid  if possible.

**Comment by kenzierocks** [comment]
That's true, we can just use reflection then.

**Comment by TomyLobo** [comment]
excuse the uneducated question: what is AT?

**Comment by clienthax** [comment]
@zml normal mods can use AT's but not class transformers

**Comment by kenzierocks** [comment]
@TomyLobo access transformers, can modify modifiers.

**Comment by TomyLobo** [comment]
modifying modifiers at runtime? well, we've been removing "private" and "final" modifiers from fields for a long time, without the need for a core mod

**Comment by kenzierocks** [comment]
This does it without reflection, when the bytecode is loaded I believe.

**Comment by TomyLobo** [comment]
Which doesn't help, since you still have to use reflection, no?

**Comment by kenzierocks** [comment]
Nope! ForgeGradle applies them in development so you can compile against the field.

**Comment by TomyLobo** [comment]
So if all plugins have to be aware of the fact that any other plugin may make any field public, why don't they just make everything public when installing forge, with something like that plugin I linked?
Why have people make lists of fields to make public?

**Comment by sk89q** [comment]
Various libraries do special things with public fields so it's potentially dangerous.

**Comment by TomyLobo** [comment]
good point.
anyway, sorry for putting you off topic so much :)

**Comment by Lunatrius** [comment]
You don't need a core mod, you can use FMLAT instead.

**Comment by kenzierocks** [comment]
No longer needed, 5b45c7cd9e789a3a65b786bae7b2554968bdba53.

**Pull request closed without merging**